src/ctim/schemas/openc2_network.clj

```diff
@@ -0,0 +1,117 @@
+(ns ctia.schemas.openc2-network
+  (:require [schema.core :as s]
+            [ctia.schemas.openc2vocabularies :as openc2v]))
+
+
+
+
+(s/defschema BGPBlackhole
+  {:type (s/enum "BGPBlackhole")
+   :host s/Str
+   })
```

Comment:
In before @AlexBaranosky tells you not to dangle your closing braces and parens :)